util/src/main/java/tech/pegasys/teku/util/config/NetworkDefinition.java

```diff
@@ -45,6 +45,15 @@
                   .eth1Endpoint("https://goerli.prylabs.net")
                   .startupTimeoutSeconds(120)
                   .build())
+          .put(
+              "altona",
+              builder()
+                  .constants("altona")
```

Comment:
Have you actually tried running this?  I suspect you'll need to list `altona` as a valid constants option in `Constants.createInputStream`

Reply:
Did that and ran it just now.